bin/full_conversion.py

```diff
@@ -0,0 +1,49 @@
+
+import argparse
+import ROOT
+
+def main():
+    parser = argparse.ArgumentParser(description='Convert detector')
+    parser.add_argument('-c', '--compact', help='Compact file location(s)',
+                        required=True, type=str, nargs='+')
+    parser.add_argument('-o', '--out', help='Converted file path',
+                        default='nothing', type=str)
```

Comment:
```suggestion
    parser.add_argument('-o', '--out', help='Converted file path',
                        default='', type=str)
```
Unlikely in this case but people might want to use whatever magical string that you put here. You can then no longer differentiate between that magic string and an actual user value. It's easier to deal with an empty string here, because an empty output file doesn't make sense.